presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -131,9 +131,13 @@ Accessors
 
 .. function:: ST_Area(Geometry) -> double
 
-    Returns the area of a polygon using Euclidean measurement on a two dimensional
+    Returns the area of a geometry using Euclidean measurement on a two dimensional
     plane (based on spatial ref) in projected units.
```

Comment:
This is a pre-existing issue, but would you drop "(based on spatial ref) in projected units". This wording may suggest the the functions support spatial references, but they don't.